extension/src/test/suite/experiments/index.test.ts

```diff
@@ -172,6 +173,12 @@ suite('Experiments Test Suite', () => {
   })
 
   describe('handleMessageFromWebview', () => {
+    afterEach(() => {
```

Comment:
I'm not sure why but using `after` instead of `afterEach` [threw an error on testing](https://github.com/iterative/vscode-dvc/actions/runs/3276627871/jobs/5393409165#step:8:211)

Reply:
probably not imported

Reply:
> probably not imported

Whoops, forgot to import 😅